src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -408,10 +415,10 @@ export default memo(function LoadoutBuilder({
             selectedStore={selectedStore}
             lbDispatch={lbDispatch}
             statOrder={statOrder}
-            enabledStats={enabledStats}
+            statConstraints={statConstraints}
             modStatChanges={result.modStatChanges}
             loadouts={loadouts}
-            params={params}
+            params={loadoutParameters}
```

Comment:
An upcoming change plumbs the whole loadout through this, so we can always create new loadouts "based on" the original full loadout.